daml-lf/spec/daml-lf-1.rst

```diff
@@ -2310,7 +2401,11 @@ closure of ``<ₜ``.
 Expression evaluation
 ~~~~~~~~~~~~~~~~~~~~~
 
-Daml-LF evaluation is only defined on closed, well-typed expressions.
+Daml-LF evaluation is only defined on well-typed expressions that
+contain no free term-level variables, and any type-level variables
+must be of an erasable kind. For the sake of brevity, and the fact
+that erasable kinds cannot affect evaluation results, we do not
+include the variable context ``Γ`` in the rules below.
```

Comment:
This is an attempt to fix the line without making too many changes. The existence of "foralls with erasable kinds" means we can't look only at closed terms, but the concept of "well-typed term" doesn't make sense unless the term is closed or you are given a context Γ in which the term is considered well-typed. But rather than adding Γ everywhere, we can just say we're eliding it, because it takes up space and isn't relevant to the evaluation outcome.